integration/tests/git_source/git_source_repeated_remote.rb

```diff
@@ -0,0 +1,68 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'Verify the same remote can be used in more than one object'
+
+env_path = on(master, puppet('config print environmentpath')).stdout.rstrip
+r10k_fqp = get_r10k_fqp(master)
+git_environments_path = '/root/environments'
+git_repo_path = '/git_repos'
+git_repo_name = 'environments'
+git_control_remote = File.join(git_repo_path, "#{git_repo_name}.git")
+code_dir = '/etc/puppetlabs/code/environments/production'
```

Comment:
Should the `env_path` be incorporated into this somehow? Is it `"#{env_path}/production"` or `"/etc/puppetlabs/code/#{env_path}/production"`? Looks like the former, based on https://puppet.com/docs/puppet/5.5/environments_creating.html#environmentpath .

Reply:
The `env_path` is the absolute path, so `/etc/puppetlabs/code/environments`. But yeah it can be incorporated.

Reply:
cool, yeah, I think if we include `env_path`, it's clearer that the `git_repo_name` has no relationship to that path.

Reply:
updated